### DISCUSSION TITLE
poroMechanicalFoam - extension of solids4Foam for variably saturated porous media

### DISCUSSION THREAD

**Discussion by denMaier** [solids4foam/solids4foam#141] [Show and tell]
Hey all,

I finally managed to make my project public! Its called poroMechanicalFoam.
The aim is to extend solid4Foam capabilities to handle coupled flow-deformation in porous media, especially for variably saturated cases. However, it also provides a few additions to saturated porous media (compared to the porous solver in s4F) and quite a few quality of life tools for simulating porous media.

Here the link to the github-repo: https://github.com/baw-de/poroMechanicalFoam/tree/main

**Comment by philipcardiff** [comment]
Very nice. Congratulations Denis!
